sway/input/cursor.c

```diff
@@ -212,6 +212,27 @@ void cursor_rebase_all(void) {
 	}
 }
 
+void cursor_update_image(struct sway_cursor *cursor,
+		struct sway_node *node) {
+	if (node && node->type == N_CONTAINER) {
+		// Try a node's resize edge
+		enum wlr_edges edge = find_resize_edge(node->sway_container, NULL, cursor);
```

Comment:
Note for reviewer: the original code had

```c
enum wlr_edges edge = find_resize_edge(node->sway_container, surface, cursor);
```

...but following the control flow, `surface` was guaranteed to always be `NULL` at that point.